include/gridtools/communication/low_level/proc_grids_3D.hpp

```diff
@@ -181,23 +181,23 @@ namespace gridtools {
             int _coords[3];
 
             if (m_cyclic.value(0))
```

Comment:
What does `m_cyclic` mean?

Reply:
Whether it should be a cyclic communicator. If the communicator is not cyclic, we don't need to communicator; so we don't need the fix in the else case.